### PR TITLE
fix: parse chainhooks satpoint to get offset

### LINKED ISSUE #54: API gives a wrong offset value for some ordinals
For some ordinals, API gives a big offset value like `435509929`, while ordinals.com shows offset `0`.

Example:
https://api.hiro.so/ordinals/v1/inscriptions/23198d6656ae23a9313eb3c2fffc8e6179e95905b92a042813feb705b6aeb1c5i0
https://ordinals.com/inscription/23198d6656ae23a9313eb3c2fffc8e6179e95905b92a042813feb705b6aeb1c5i0

A lot more ordinals like this are in this wallet: https://api.hiro.so/ordinals/v1/inscriptions?address=bc1ppmhxm8myvy55547ee4gyjv5ewzms4hrnhvpfg2z4el68as5f65vsskhg0y

It seems that the issue impacts only ordinals that haven't been transferred yet, i.e. `tx_id == genesis_tx_id`

### PR DESCRIPTION
**Pull request by rafaelcr**
Fixes #54

### CODE REVIEW

**Comment by markmhx** [comment]
@rafaelcr curious if this fix is ready and just awaiting review? Any way it can get expedited, since it seems to be impacting all new inscriptions for users in the wallet and preventing them from sending? 

cc @314159265359879

**Comment by 314159265359879** [comment]
Thanks for building a fix for this Rafael, quiet a few wallet users are looking forward to the fix for #54

File: src/pg/pg-store.ts [hunk 1]
```diff
@@ -98,12 +104,12 @@ export class PgStore extends BasePgStore {
...
                   block_hash,
+                  block_height,
                   tx_id,
                   genesis_id: reveal.inscription_id,
-                  block_height: event.block_identifier.index,
                   address: reveal.inscriber_address,
-                  output: `${tx_id}:0`,
-                  offset: reveal.ordinal_offset.toString(),
+                  output: `${satpoint.tx_id}:${satpoint.vout}`,
+                  offset: satpoint.offset ?? null,
```
Comment (lgalabru) [inline src/pg/pg-store.ts:112]: null or 0?

Comment (rafaelcr) [reply src/pg/pg-store.ts:112]: IIRC chainhooks returns this as null if the satpoint gets spent as fees? Or do I have that wrong?

**Review by lgalabru — APPROVED** [review]
LGTM! Just one feedback on the `null` / `0` handling.

File: src/pg/pg-store.ts (hunk 1 above)
Comment (lgalabru) [reply src/pg/pg-store.ts:112]: Oh you're right, perfect. That works, but we'll revisit this behavior chainhook side. Thanks!

**Pull request merged**